lib/lighthouse/facilities/response.rb

```diff
@@ -0,0 +1,36 @@
+# frozen_string_literal: true
+
+require 'common/models/base'
+
+module Lighthouse
+  module Facilities
+    class Response < Common::Base
+      attribute :body, String
+      attribute :status, Integer
+      attribute :parsed_json, String
+      attribute :data, Object
+
+      def initialize(body, status)
+        self.body = body
+        self.status = status
+
+        self.parsed_json = parse_json
+        self.data = parsed_json['data']
+      end
+
+      def parse_json
```

Comment:
ok, Why?
this doesn't get reused and while I do like putting work in methods this is a one-liner your abstracting that gets used as a one-liner in most of the app.

Reply:
This was a bit of copypasta. I have simplified this init.